Modules/gcmodule.c

```diff
@@ -221,15 +222,25 @@ Return the list of objects that directly refer to any of 'objs'.
 [clinic start generated code]*/
 
 static PyObject *
-gc_get_referrers_impl(PyObject *module, PyObject *args)
-/*[clinic end generated code: output=296a09587f6a86b5 input=bae96961b14a0922]*/
+gc_get_referrers_impl(PyObject *module, Py_ssize_t nargs,
+                      PyObject *const *args)
+/*[clinic end generated code: output=1d44a7695ea25c40 input=bae96961b14a0922]*/
 {
-    if (PySys_Audit("gc.get_referrers", "(O)", args) < 0) {
+    PyObject *varargs = _PyTuple_FromArray(args, nargs);
+
+    if (!varargs) {
+        return NULL;
+    }
+    if (PySys_Audit("gc.get_referrers", "(O)", varargs) < 0) {
```

Comment:
BTW, did you try replacing `"(O)"` with `"O"` and moving the `PySys_Audit` call back to the start of the function?